extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx

```diff
@@ -138,16 +139,23 @@ function PanelStudyBrowserTracking({
       DisplaySetService.EVENTS.DISPLAY_SETS_ADDED,
       newDisplaySets => {
         newDisplaySets.forEach(async dSet => {
+          const displaySetInstanceUID = dSet.displaySetInstanceUID;
+
           const newImageSrcEntry = {};
           const displaySet = DisplaySetService.getDisplaySetByUID(
-            dSet.displaySetInstanceUID
+            displaySetInstanceUID
           );
+
+          if (dSet.madeInClient) {
```

Comment:
I'm not sure I want to expand `displaySet`'s props to support this specifically. If there are other benefits, we can talk about keeping it?

Reply:
Changed, now on event.